app/code/Magento/UrlRewriteGraphQl/etc/schema.graphqls

```diff
@@ -1,16 +1,16 @@
 # Copyright © Magento, Inc. All rights reserved.
 # See COPYING.txt for license details.
 
+type Query {
+    urlResolver(url: String!): EntityUrl @resolver(class: "Magento\\UrlRewriteGraphQl\\Model\\Resolver\\EntityUrl") @doc(description: "The urlResolver query returns the relative URL for a specified product, category or CMS page")
+}
+
 type EntityUrl @doc(description: "EntityUrl is an output object containing the `id`, `canonical_url`, and `type` attributes") {
```

Comment:
It seems to me that you left 'canonical_url' in the description. Please update

Reply:
Okay updating

Reply:
@TomashKhamlai updated